actors/builtin/miner/deadline_state_test.go

```diff
@@ -162,64 +157,61 @@ func TestDeadlines(t *testing.T) {
 				bf(1, 2, 3, 4),
 				bf(5, 6, 7, 8),
 				bf(9),
-			).assert(t, rt, dl)
+			).assert(t, store, dl)
 	}
 
 	// Test the basic scenarios (technically, we could just run the final one).
 
 	t.Run("adds sectors", func(t *testing.T) {
-		rt := builder.Build(t)
-		dl := emptyDeadline(t, rt)
-		addSectors(t, rt, dl)
+		store := adt.AsStore(builder.Build(t))
```

Comment:
It would be even better if we could avoid instantiating the mock runtime at all!

Try `NewADTStore` in `support/ipld.go`